fietspaaltjes.map

```diff
@@ -0,0 +1,46 @@
+#==============================================================================
+#
+# fietspaaltjes.map
+# doel: serveren locaties fietspaaltjes
+#
+#==============================================================================
+
+MAP
+    NAME "fietspaaltjes"
+    INCLUDE "header.inc"
+    DEBUG 1
+
+    WEB
+      METADATA
+        "ows_title"           "fietspaaltjes"
+        "ows_abstract"        "fietspaaltjes"
+        "wms_extent"          "100000 450000 150000 500000"
+      END
+    END
+
+    LAYER
+        NAME        "fietspaaltjes"
+        TYPE        POINT
+        INCLUDE     "connection_dataservices.inc"
+        DATA        "geometry from public.fietspaaltjes_fietspaaltjes USING srid=28992 USING UNIQUE id"
+
+        CLASS
+            STYLE
+                COLOR 200 50 50
+                ANTIALIAS TRUE
+                SYMBOL 'stip'
+                SIZE 10
+            END
+        END
+        METADATA
+          "wms_enable_request"  "*"
+          "wms_group_title"     "Buurten"
+          "wms_abstract"        "Aardgasvrij status per buurt"
```

Comment:
Ik zou toch de referentie naar de aardgasvrij kaart  veranderen :)

Reply:
oeps, is nu aangepast.

Reply:
WFS, goed punt, ook de wfs toegevoegd.